payload/reggae/options.d

```diff
@@ -45,6 +45,7 @@ struct Options {
     bool perModule;
     bool allAtOnce;
     bool isDubProject;
+    bool isCmakeProject;
```

Comment:
This would be the case if the entire project was CMake right? But the idea (at least my idea) was to add cmake projects to a build.